logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/TurnLoudSpeakerOnUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.logic.feature.call.usecase
+
+import com.wire.kalium.logic.feature.call.MediaManagerService
+
+class TurnLoudSpeakerOnUseCase(private val mediaManagerService: MediaManagerService) {
+    operator fun invoke() {
```

Comment:
A bit late to the parteey, but wouldn't it be simpler to just have one use case called `EnableLoudspeakerUseCase` that would receive a `val shouldEnable: Boolean` when invoked?

Reply:
Yes it could be done like.. just I wanted to keep `TurnLoudSpeakerOnUseCase` responsible for only one thing which is muting call